contracts/modules/cover/Cover.sol

```diff
@@ -637,6 +637,18 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     }
   }
 
+  function editProductTypes(
+    uint[] calldata productTypeIds,
+    ProductType[] calldata newProductTypes,
+    string[] calldata ipfsMetadata
+  ) external override onlyAdvisoryBoard {
+    
+    for (uint i = 0; i < productTypeIds.length; i++) {
+      _productTypes[productTypeIds[i]] = newProductTypes[i];
```

Comment:
`ProductType` contains `claimMethod` and `gracePeriodInDays`. I don't think it's okay to change the claim method, hence instead of passing a `ProductType[]` we could only pass a `uint[]` with new grace periods.